packages/uploadthing/package.json

```diff
@@ -85,7 +86,8 @@
   "eslintConfig": {
     "root": true,
     "extends": [
-      "@uploadthing/eslint-config/base"
+      "@uploadthing/eslint-config/base",
+      "plugin:@uploadthing/recommended"
```

Comment:
it didn't find the plugin if i put it in the @uploadthing/eslint-config sadly :/

Reply:
Huh, weird. 

Maybe: 
https://github.com/microsoft/rushstack/tree/main/eslint/eslint-patch

Reply:
seems to do the trick!